src/index.jsx

```diff
@@ -49,27 +55,77 @@ function Form() {
         setPref('app', app);
     }, [app])
 
+    const menuItems = Object.entries(AVAILABLE_APPS).map(([name, value]) => ({
+        text: value.title,
+        key: name
+    }));
+    
+    const handleMenuChange = (item) => {
+        if (item) {
+            setApp(item.key);
+        }
+    };
+
+    const changeOpenState = (item) => {
+        if(isDropdownOpen == true){
+            document.getElementById("root").className = "normal";
```

Comment:
I see this is primarily being used to set height / width. In general my understanding of CSS is that only one of these needs to be set (likely `root`, rather than `popoutBody`). The parent element should expand to take the appropriate height / width.

Also, use [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) instead of `className`. `className` will overwrite any other classes this may get in the future.

I would also suggest that 'normal' not be a class, but just be the lack of classes. So CSS can have something for `#root`, and then a different style for `#root.expanded`.

Reply:
Disregard this, and see my comment below about getting rid of the `Dropdown` component entirely so we don't have to do this fiddly bit.